spec/index.html

```diff
@@ -495,6 +496,26 @@ <h2>RDF Documents and Syntaxes</h2>
     and still the subject of group discussion. An alternative to conformance
     levels, "profiles", may be adopted instead, abandoned, or described in 
     another specification.</p>
+
+  <section id="rdf-strings">
+    <h2>Strings in RDF</h2>
+
+    <p>RDF uses Unicode [[Unicode]] as the fundamental representation for string values.
+      Within this, and related specifications, the term <dfn id="dfn-rdf-string">string</dfn>
+      is used to describe an ordered sequence of zero or more <a data-lt="code point" class="lint-ignore">Unicode code points</a>
+      in any <a data-cite="i18n-glossary#dfn-character-encoding">character encoding</a>.
+      Note that most <a>concrete RDF syntaxes</a> require the use
+      of the UTF-8 character encoding [[!RFC3629]], 
+      which also prevents the use of certain non-character values.
```

Comment:
They can be inserted using `\u` and `\U`. This sentence should be removed.

```suggestion
```

Reply:
Might it be better amended? Perhaps as --
```suggestion
      Note that most <a>concrete RDF syntaxes</a> require the use
      of the UTF-8 character encoding [[!RFC3629]], 
      which requires that certain non-character values be expressed
      using the `\u0000` or `\U00000000` forms.
```